WebRoot/css/zm.css

```diff
@@ -3183,28 +3183,58 @@ HTML>BODY .appt-selected .appt_allday_body
  * Reminder Dialog
  */
 
+.ZmReminderDialogContainer > div.WindowOuterContainer {
+	@ReminderDialogContainer@
+}
+
+.ZmReminderDialogContainer .DwtDialogTitle {
+	@ReminderDialogTitle@
+}
+
+.ZMReminderDialogContainer .WindowInnerContainer {
+	@ReminderDialogInnerContainer@
+}
+
+/*Content along with action buttons*/
 .ZmReminderDialog {
-	overflow:hidden;
-	width:auto; 			/*??? SIZE */
-	@BoxMargin@
-	margin-bottom:0px;
+	@ReminderDialogFullContent@
+}
+
+.ZmReminderDialog .horizSep {
+	@ReminderDialogSeparator@
+}
+
+/* Content without bottom action buttons*/
+.ZmReminderDialogContent > table {
+	@ReminderDialogContentTable@
+}
+
+.ZmReminderDialogContent {
+	@ReminderDialogContent@
+}
+
+/* Individual appointment snooze/dismiss buttons */
+.ZAppointmentReminderButton {
+	@ReminderDialogAppointmentButton@
+}
+
+.ZmReminderDialog .ZAppointmentReminderButton .ZButtonTable {
+	@DialogButton@
 }
 
 /* Overdue appt text */
 .ZmReminderOverdue {
 	@Text-important@
-	font-weight:bold;
 }
 
 /* Appt coming soon text */
 .ZmReminderSoon {
 	@Text-important@
-	font-weight:bold;
 }
 
 /* Appt somewhere in the future */
 .ZmReminderFuture {
-
+	@Text-important@
```

Comment:
Can we use a variable like ReminderDialogFutureText?

Reply:
Updated here: c9b7132cc36aeef41fa94b4a5d2febfda45b691a
Thanks.